neon-bindings/neon
# fix: `neon new` not exiting on windows

Fixes #406
Closes https://github.com/neon-bindings/neon/pull/501

This bug fixes a bug where running `neon new` on windows will hang without ever exiting despite creating a new repo and all.

## Changes
* Migrates to use native fs module
* Bumps TS to latest version
* Adds `make-promises-safe` to CLI entrypoint to ensure promise do not silently fail
* Build targets ES6
* Removes node 8 from CI, adds node 14
* Removes `ts-unknown` and `es6-polyfill` dependencies
* Change methods to sync

## Testing this PR
Make sure you're on a windows machine to test this PR
```bash
git clone https://github.com/neon-bindings/neon test-repo
cd test-repo
npm i
npm run transpile
node bin/cli.js new foo
```

This fix is based on @usagi 's PR (https://github.com/neon-bindings/neon/pull/501). Thanks!
